tools/tune/common/tplg_write.m

```diff
@@ -15,6 +15,11 @@ function tplg_write(fn, blob8, name, comment, howto)
 
 %% Write blob
 fh = fopen(fn, 'w');
+if fh < 0
```

Comment:
Changes to common could be in another PR maybe? I realized I had in another PR similar error adds to other exports.

For now I'm not moving tools/tune/common -- I don't know where they could go, or is there any benefit to move the helper functions closer to src/audio modules.